docker/hadoop/common/conf-hive-kerberos/hive-site.xml

```diff
@@ -45,6 +45,10 @@
         <name>hive.users.in.admin.role</name>
         <value>hdfs,hive</value>
     </property>
+    <property>
```

Comment:
does this do anything? It just seems to be moved

Reply:
yes, just moved to be in the same place for both config files